src/UnitTests/MessageShovelTests.cs

```diff
@@ -130,7 +130,7 @@ static async Task<TransferDetails> Transfer(
 
         var targetEndpoint = new BridgeEndpoint("TargetEndpoint", targetAddress);
         var dispatcherRegistry = new FakeTargetEndpointRegistry("TargetTransport", targetEndpoint);
-        var shovel = new MessageShovel(logger, dispatcherRegistry);
+        var shovel = new MessageShovel(logger, dispatcherRegistry, new FakeFinalizedBridgeConfiguration());
```

Comment:
Do we need the interface - can't we just use the class?

```suggestion
        var shovel = new MessageShovel(logger, dispatcherRegistry, new FinalizedBridgeConfiguration(null, true));
```

Reply:
I like working with interfaces, but I've removed it so that there are less changes to the code